packages/gatsby-plugin-offline/src/gatsby-browser.js

```diff
@@ -40,8 +39,14 @@ exports.onServiceWorkerActive = ({
     )
   )
 
-  serviceWorker.active.postMessage({
-    api: `gatsby-runtime-cache`,
-    resources: [...resources, ...prefetchedResources],
+  const resources = [...headerResources, ...prefetchedResources]
+  resources.forEach(resource => {
+    // Create a prefetch link for each resource, so Workbox runtime-caches them
+    const link = document.createElement(`link`)
```

Comment:
Isn't it possible that we're duplicating a prefetch here? i.e. `head > link[href]` could match

```
<link rel="prefetch" href="/static/d/368/path---showcase-c-41-46f-ykWCDEvUZCguGaUYZsNI5k1Io.json">
```

and we'd create another prefetch link. Is that OK?

Reply:
Yes, this is expected behaviour - we need to "prefetch" after the SW has activated, so that Workbox can cache the responses.